content/en/ddsql_reference/ddsql_default.md

```diff
@@ -219,23 +234,23 @@ SELECT
 ### `TRIM`
 {{< code-block lang="sql" >}}
 SELECT
-  TRIM(name) AS trimmed_name
+  trim(name) AS trimmed_name
```

Comment:
Nit: Since SQL is not case sensitive,  it is find to make this lowercase. However, it is nice to keep them upper case as they are functions/keywords